src/main/scala/org/clulab/asist/agents/DacClient.scala

```diff
@@ -50,82 +45,108 @@ class DacClient (
   // json
   implicit val formats = org.json4s.DefaultFormats
 
-  // shutdow the actor system
-  def shutdown(): Unit = {
-    system.terminate()
-  }
-
   // schedule the next iteration after reseting the DAC server
   def resetServer(s: RunState): Unit = {
-    logger.info("Resetting DAC server at http://localhost:8000")
+    logger.info("Resetting DAC server at {}",serverLocation)
```

Comment:
any reason you're not using ```s"Resetting DAC Server at ${serverLocaion}"``` ?

Reply:
I thought the '{}',value was more idiomatic and easier to read.